e2e/packages/sync-test/registerDelegationWithSignature.test.ts

```diff
@@ -0,0 +1,110 @@
+import { afterEach, beforeEach, describe, expect, it } from "vitest";
+import type { ViteDevServer } from "vite";
+import { Browser, Page } from "@playwright/test";
+import { createAsyncErrorHandler } from "./asyncErrors";
+import { deployContracts, startViteServer, startBrowserAndPage, openClientWithRootAccount } from "./setup";
+import { rpcHttpUrl } from "./setup/constants";
+import { waitForInitialSync } from "./data/waitForInitialSync";
+import { createBurnerAccount, resourceToHex, transportObserver } from "@latticexyz/common";
+import { http, createWalletClient, ClientConfig } from "viem";
+import { mudFoundry } from "@latticexyz/common/chains";
+import { encodeEntity } from "@latticexyz/store-sync/recs";
+import { callPageFunction } from "./data/callPageFunction";
+import worldConfig from "@latticexyz/world/mud.config";
+import { worldToV1 } from "@latticexyz/world/config/v2";
+import { types } from "@latticexyz/world/internal";
+import { getWorld } from "./data/getWorld";
+import { callRegisterDelegationWithSignature } from "./data/callRegisterDelegationWithSignature";
+
+const PRIVATE_KEY = "0x67bbd1575ecc79b3247c7d7b87a5bc533ccb6a63955a9fefdfaf75853f7cd543";
```

Comment:
should be more specific with the naming for what this constant is used for, maybe `BURNER_WALLET_PRIVATE_KEY`?